cmd/influxd/recovery/auth/auth.go

```diff
@@ -197,12 +217,20 @@ func PrintAuth(ctx context.Context, w io.Writer, v []*influxdb.Authorization, us
 		if err == nil && user != nil {
 			userName = user.Name
 		}
+		var token string
+		if t.Token != "" {
+			token = t.Token
+		} else if t.HashedToken != "" {
+			token = "REDACTED"
```

Comment:
Could "N/A" and "REDACTED" be Enums or consts?

Reply:
This will be turned into constants on the next push.